### ISSUE TITLE
Object Detection API : Feature request for have weighted softmax in order to work with poorly perform classes . 

### ISSUE THREAD

**Issue by shamanez** [tensorflow/models#2248]
In object detection sometimes there are few classes with low validation accuracy .  If we can use weighted softmax or score function I believe we can give more attention to poorly perform classes lost when doing the gradient upgrade .

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
